core/field_textinput.js

```diff
@@ -362,9 +362,11 @@ Blockly.FieldTextInput.prototype.bindInputEvents_ = function(htmlInput) {
 Blockly.FieldTextInput.prototype.unbindInputEvents_ = function() {
   if (this.onKeyDownWrapper_) {
     Blockly.unbindEvent_(this.onKeyDownWrapper_);
+    this.onKeyDownWrapper_ = null;
   }
   if (this.onKeyInputWrapper_) {
     Blockly.unbindEvent_(this.onKeyInputWrapper_);
+    this.onKeyInputWrapper_ = null;
   }
   if (this.onBlurInputWrapper_) {
     Blockly.unbindEvent_(this.onBlurInputWrapper_);
```

Comment:
Missed one?

Reply:
``onBlurInputWrapper_``

Reply:
Haha I think this may have appeared when I rebased, will fix!